### PR TITLE
Add floating button

### PR DESCRIPTION
**Pull request by berlinum**
Add floating button to Storybook.

__Preview__: https://bricks-pipel-floating-6r7owda0.herokuapp.com/storybook/index.html?path=/story/button--animated-floating-button

Screenshot 2020-04-16 at 23 40 27

### CODE REVIEW

File: client/src/stories/FloatingButton.stories.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react';
+import { FloatingButton } from '../components/FloatingButton';
+
+export default {
+  title: 'Button',
+  component: FloatingButton,
+};
+
+export const AnimatedFloatingButton = () => {
+  const [active, setActive] = useState(false);
```
Comment (florianGierlichs) [inline client/src/stories/FloatingButton.stories.js:10]: Maybe you can outsource the state into the component file.
With that, you could have a very clean component appeal with only the components after each other. 
Think about a complex page where you have to call a lot of components. Either you have every component and for every component additional lines of code where you set the states OR you put all states into the component itself and when you call the component, you only have 1 line of code, the component itself.

File: client/src/components/FloatingButton.js [hunk 2]
```diff
@@ -12,13 +14,20 @@ export const AddButton = styled.button`
   background-repeat: no-repeat;
   background-position: center;
   box-shadow: 2px 2px 6px rgba(0, 0, 0, 0.4);
+  transform: rotate(${(props) => (props.active ? 45 : 0)}deg);
```
Comment (marwinburesch) [inline client/src/components/FloatingButton.js:17]: maybe one more turn? 🦥

File: client/src/stories/FloatingButton.stories.js (hunk 1 above)
Comment (marwinburesch) [reply client/src/stories/FloatingButton.stories.js:10]: it really depends on how the button is used, I think for now it makes the most sense how it is right now

File: client/src/components/FloatingButton.js (hunk 2 above)
Comment (berlinum) [reply client/src/components/FloatingButton.js]: Done! ;)

**Pull request merged**